packages/metal-quartz-icon/demos/index.html

```diff
@@ -9,32 +9,57 @@
 </head>
 <body>
 	<h1>Metal Quartz Icon</h1>
-
 	<div id="quartzIcons"></div>
 
+	<h1>Glyphicons</h1>
+	<div id="glyphicons"></div>
+
+	<h1>Lexicon Fa</h1>
+	<div id="lexiconFa"></div>
```

Comment:
So far it hasn't been that hard to learn (it helps to have all the other metal modules to look at), it's just a lack of metal documentation that increases the learning curve a bit. I don't think it's possible to decouple Bootstrap from these modules since we aren't including any css. The only way it makes sense is if we remove Bootstrap from Quartz.

I can change this to Quartz Fa.